index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="//use.fontawesome.com/releases/v5.0.7/css/all.css"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>My Portfolio Project</title>
+  </head>
+  <body>
+    <main>
+      <!--toolbar section-->
+      <header id="page-head">
+        <nav id="navbar">
+          <div id="nav-header">
+            <div id="nav-title">
+                <a href="#welcome-section" id="title">Welcome</a>
+            </div>
```

Comment:
- [ ] please as part of the requirements, the logo should be part of the header but not in the `nav`